apps/src/music/views/MusicView.jsx

```diff
@@ -33,8 +33,8 @@ import {
   setSelectedTriggerId,
   clearSelectedTriggerId,
 } from '../redux/musicRedux';
-import KeyHandler from './KeyHandler';
-import Callouts from './Callouts';
+// import KeyHandler from './KeyHandler';
+// import Callouts from './Callouts';
```

Comment:
Should these commented out lines in this file be removed? Or are they going to be turned back on soon and the disabling is temporary?